README.md

```diff
@@ -11,10 +11,14 @@ This tool automates deployment of Dynatrace Monitoring Configuration to one or m
 and [Configuration Structure](#configuration-structure)
 
 - [Dynatrace Monitoring as Code](#dynatrace-monitoring-as-code)
-  - [Using Monitoring as Code Tool](#using-monitoring-as-code-tool)
     - [Install Monaco](#install-monaco)
+      - [On Mac or Linux systems perform the following](#on-mac-or-linux-systems-perform-the-following)
+      - [On Windows](#on-windows)
     - [Commands (CLI)](#commands-cli)
       - [Dry Run (Validating Configuration)](#dry-run-validating-configuration)
+      - [Experimental new CLI](#experimental-new-cli)
```

Comment:
Out of curiosity: Do you generate the table of contents? :thinking:

Reply:
Yes I do, I would not catch these changes otherwise :D 

[Markdown All In One vscode extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) (I use vscode as my go to markdown editor)